IdentityCore/src/broker_operation/request/MSIDBrokerOperationRequest.m

```diff
@@ -69,7 +69,7 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)er
         _clientAppName = [json msidStringObjectForKey:MSID_BROKER_CLIENT_APP_NAME_KEY];
         
         NSString *uuidString = [json msidStringObjectForKey:MSID_BROKER_CORRELATION_ID_KEY];
-        _correlationId = [[NSUUID alloc] initWithUUIDString:uuidString];
+        self.correlationId = [[NSUUID alloc] initWithUUIDString:uuidString];
```

Comment:
initializers should use instance variables (so _correlationId instead of self.correlationID)

Reply:
Before the PR:
2 Classes `MSIDBrokerOperationRequest` & `MSIDBrokerOperationBrowserTokenRequest` extends `MSIDBaseBrokerOperationRequest`. Both child classes have  `correlationId` => I think we can move the `correlationId` to the father class (`MSIDBaseBrokerOperationRequest`)
But the base class doesn't have the **`init`** method yet. So I think it'd make sense if we create the **`init`** method of base class and inside that method we create the `correlationId` so the child class doesn't need to deal with that logic.

Reply:
I searched for it a little bit and found they suggested to add that property in protected area of father class (`MSIDBaseBrokerOperationRequest`), so the subclass (`MSIDBrokerOperationBrowserTokenRequest`) can use `_correlationId` in its init method. So it will look like:
```
@interface MSIDBaseBrokerOperationRequest : NSObject
{
    @protected
    NSUUID *_correlationId;
}
@property (nonatomic) NSUUID *correlationId;
```
Will it be better :D

Reply:
yes, this would work.

Reply:
Thank you. Address in next commit